trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/plugins/generator/cacsl2boogietranslator/witness/ExtractedFunctionContract.java

```diff
@@ -77,9 +76,15 @@ private ACSLNode parseString(final ILocation loc, final String string) {
 		}
 	}
 
-	public Stream<ACSLNode> transformToAcsl() {
+	public List<ACSLNode> getAcslContracts() {
```

Comment:
_minor:_ should this be `getAcslContractClauses` or similar? I was confused by the idea that there would be multiple contracts.